package.json

```diff
@@ -1,50 +1,45 @@
 {
   "name": "react-page-visibility",
-  "version": "0.0.2",
+  "version": "1.0.0",
   "description": "Declarative, nested, stateful, isomorphic page visibility for React",
   "author": "Gilad Peleg <giladp007@gmail.com> (http://giladpeleg.com)",
   "repository": "pgilad/react-page-visibility",
-  "keywords": "react, components, visibility, page, higher-order, browser",
-  "main": "index.js",
+  "keywords": "react, components, visibility, page, higher-order, browser, react-component",
+  "main": "dist/index.js",
+  "module": "src/index.js",
   "files": [
-    "index.js",
-    "visibility.js"
+    "dist",
+    "src"
   ],
   "engines": {
     "node": ">=6.0.0"
   },
   "scripts": {
-    "build": "webpack",
-    "test": "tape -r ./test/babel-register-setup test/*-test.js | faucet",
-    "test-browser": "npm run build && cat ./build/bundle.js | tape-run -r faucet",
-    "test-ci": "npm run build && zuul --no-coverage --browser-output-timeout 15000 --concurrency 1 -- ./build/bundle.js",
-    "test-local": "npm run build && zuul --local -- ./build/bundle.js",
-    "test-phantom": "npm run build && zuul --phantom -- ./build/bundle.js"
+    "build": "NODE_ENV=production babel src -d dist",
+    "clean": "rm -rf dist",
+    "lint": "eslint index.js",
```

Comment:
The `index.js` file is no longer in the project so you probably want to just add `src/` or `.` (if you want to eslint tests also)